db/questions/Databases.txt

```diff
@@ -0,0 +1,5 @@
+2
+Official CFAR Questions
+appliedrationality.org/uploads/games/credence/OfficialCfarQuestions.xml
+LessWrong Berlin Meetup Questions
+appliedrationality.org/uploads/games/credence/LW_Berlin.xml
```

Comment:
Feels like this belongs in the README.md or a Creedence-game-specific markdown file as well.